code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -262,6 +262,10 @@
 
 
 /obj/structure/closet/MouseDrop_T(atom/movable/O, mob/user)
+	if(!isliving(user))
+		return
+	if(isxenohivemind(user))
+		return
```

Comment:
Line 279 will always be true fix this please

Reply:
Do you mean the old 279 or the new 279?

Reply:
Assuming you mean the if(ismob(O)) check, that won't be affected by the isliving check because it's checking for O, the target movable, while the if(!isliving(user)) is checking the user instead.